example/components/CrossSellGroup.jsx

```diff
@@ -34,9 +35,10 @@ CrossSellGroup.propTypes = {
     lg: PropTypes.number,
   }),
   someFunction: PropTypes.func,
-  crossSellCards: validateAll(customPropType(CrossSellCard)).isRequired,
-  children: childType(CrossSellCard),
-  crossSellCard: customPropType(CrossSellCard),
+  // /** @array */
+  // crossSellCards: componentWithName('CrossSellCard')).isRequired,
```

Comment:
Do we want these to be commented out?

Reply:
Mainly did this to show an example of using the airbnb-prop-types lib.. didn't want to actually import it into this library though... But if it is confusing then I can remove

Reply:
You can still have it as a dev dependency.

Reply:
oh, right. haha